versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.61"
+      version = ">= 5.70"
```

Comment:
For posterity, there appears to be an outstanding issue where [aws provider `>= 5.69` doesn't properly initialize for a large set of users](https://github.com/hashicorp/terraform-provider-aws/issues/39523). Until that is resolved, impacted users will need to back-pin this chart to [`v20.24.3`](https://github.com/terraform-aws-modules/terraform-aws-eks/releases/tag/v20.24.3) or earlier.